doc/rtd/topics/network-config-format-v2.rst

```diff
@@ -338,7 +338,7 @@ Set whether to set all slaves to the same MAC address when adding
 them to the bond, or how else the system should handle MAC addresses.
 The possible values are ``none``, ``active``, and ``follow``.
 
-**gratuitious-arp**: <*(scalar)>*
+**gratuitous-arp**: <*(scalar)>*
```

Comment:
confirmed fix-released in [netplan bug](https://bugs.launchpad.net/netplan/+bug/1756701) and in (upstream netplan source](https://github.com/canonical/netplan/blob/main/src/parse.c#L2001)